docs/index.md

```diff
@@ -0,0 +1,80 @@
+---
+organization: Turbot
+category: ["software development"]
+icon_url: "/images/plugins/turbot/pagerduty.svg"
+brand_color: "#00FF00"
+display_name: "PagerDuty"
+short_name: "pagerduty"
+description: "Steampipe plugin to query services, teams, escalation policies and more from your PagerDuty account."
+og_description: "Query PagerDuty with SQL! Open source CLI. No DB required."
+og_image: "/images/plugins/turbot/pagerduty-social-graphic.png"
+---
+
+# PagerDuty + Steampipe
+
+[PagerDuty](https://www.pagerduty.com/) is a platform for agile incident management.
+
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query cloud APIs using SQL.
+
+List services in your PagerDuty account:
+
+```sql
```

Comment:
Is a query possible that will check if there are any escalation policies/rules that do not alert anybody, i.e., are there any types of incident that will not be sent to everyone, or anything similar? I think this type of query would be useful to be able to quickly identify are there any gaps in coverage.

Reply:
@Subhajit97 What impact does a disabled service have in PagerDuty?